### PR TITLE
Added Get Text function to support original Selenium Library.

### PR DESCRIPTION
**Pull request by jouk0**
... it brakes my tests without it :)

### CODE REVIEW

**Comment by j1z0** [comment]
Hi Jouk0 can you add some tests around the iFrame locator functionality change?  If so I think it's OK to merge.

Thanks, 
Jeremy

**Comment by adwu73** [comment]
I don't know how to add test yet, so I will learn from Jouko this time, Tks!

Adam

File: src/Selenium2Library/keywords/_element.py [hunk 1]
```diff
@@ -228,6 +228,13 @@ def get_value(self, locator):
         See `introduction` for details about locating elements.
         """
         return self._get_value(locator)
+        
+    def get_text(self, locator):
```
Comment (j1z0) [inline src/Selenium2Library/keywords/_element.py:232]: This get_text is fine I'll go ahead and add it to the base, I'm not sure about the iFrame stuff though.  Ed / Ryan any thought on the iFrame stuff?

Comment (adwu73) [inline src/Selenium2Library/keywords/_element.py:232]: I have tested the iFrame related code in my real projects. They works fine, and the logic is very simple. Please merge into base as well, tks!

**Comment by j1z0** [comment]
Can u guys checkout the Iframes branch and see if works for u.

**Comment by qitaos** [comment]
i will try it.

**Comment by jollychang** [comment]
any updates?

**Comment by alistair-broomhead** [comment]
Any updates? I need to be able to do Get Text for some of my tests, unless anyone has a workaround

**Comment by emanlove** [comment]
@alistair-broomhead, @jollychang: I've cherry-picked out the 'get text' patch [here](https://github.com/rtomac/robotframework-selenium2library/pull/72) and am having @j1z0 review it.  Should be merged in soon.

**Comment by j1z0** [comment]
merged. :)

**Comment by j1z0** [comment]
I'm closing this issues as the cherry-picked version from @emanlove has been merged into master

**Pull request closed without merging**